docs/content/en/docs/Integrations/codimd.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "CodiMD"
+linkTitle: "CodiMD"
+weight: 31
+description: >
+    Real-time collaborative markdown editing with [CodiMD](https://github.com/hackmdio/codimd)
+---
+
+Since CodiMD is not built upon the [WOPI Protocol](https://wopi.readthedocs.io/en/latest/) (as it happens with [Collabora Code](../../iop/deployment/wopiserver)), an additional component needs to be deployed alongside the WOPI Server to fill this gap between the application provider and the application itself.
+
+## Deploying and configuring the WOPI Bridge
+
+Bridging that gap will be the job for the **WOPI Bridge**: a sidecar container to translate CodiMD API calls into file operations and vice-versa.
+
+A small configuration patch needs to be applied to an IOP deployment to enable this new component and plug it into the WOPI Server application providers. The main parameters to configure being:
+
+- The WOPI Server configuration needs to be extended to connect the WOPI Bridge via the `config.appProviders.wopibridgeurl` parameter.
+- `CODIMD_INT_URL` connects the bridge and the CodiMD service running inside the cluster.
+- The `CODIMD_EXT_URL` will be used to generate the URL presented to the end-user when `open-file-in-app-provider` is called in reva.
+
+```yaml
+wopiserver:
+  config:
+    appProviders:
+      wopibridgeurl: https://<hostname.domain.tld>/wopib
+  wopibridge:
+    enabled: true
+    env:
+      APP_ROOT: /wopib
+      CODIMD_EXT_URL: https://<hostname.domain.tld>/codimd
+      CODIMD_INT_URL: http://meshapps-codimd
+    ingress:
+      enabled: true
+      annotations:
+        kubernetes.io/ingress.class: nginx
+        nginx.ingress.kubernetes.io/ssl-redirect: "true"
+      hostname: <hostname.domain.tld>
+      path: /wopib
+```
+
+Refer to the [chart's documentation](https://artifacthub.io/packages/helm/cs3org/wopiserver#wopi-bridge-configuration) for a complete reference of all the configuration options available. Once ready, the release can be patched with the following command:
+
+```bash
+helm upgrade -i iop sciencemesh/iop --reuse-values -f wopibridge.yaml
+```
+
+## Deploying CodiMD through the `sciencemesh/meshapps` chart
+
+To deploy the application, the `sciencemesh/meshapps` umbrella chart already provides a handful of defaults for a lightweight and "_stateless_" CodiMD installation:
+
+- Based on a patched `codimd` container image (`gitlab-registry.cern.ch/authoring/notes/codimd:cernbox-integration`) with some extra features to enable this workflow.
+- Enables anonymous access, viewing and editing.
+- Disables some defaults from the vanila CodiMD installation:
+  - The registration and authentication mechanisms.
+  - The image store volume and PostreSQL disk persistence, as we will rely on the IOP Storage Providers to read and write all the content, including attached pictures.
```

Comment:
To be exact, CodiMD still has its own store + DB, but it is ephemeral (and in the future we want to deploy some automatic garbage collection for the image store, e.g. after one week). The sentence `we will rely...` is of course true.

Reply:
You're absolutely right. I've just poured some more clarity into that sentence. Thx @glpatcern 🙌